Sources/StreamChatUI/Appearance+ColorPalette.swift

```diff
@@ -45,6 +45,7 @@ public extension Appearance {
 
         public var messageCellHighlightBackground: UIColor = .streamYellowBackground
         public var pinnedMessageBackground: UIColor = .streamYellowBackground
+        public var jumpToUnreadMessagesBackground: UIColor = .streamGrayDisabledText
```

Comment:
Maybe this is a bit clear? 🤔 WDYT?
```suggestion
        public var jumpToUnreadButtonBackground: UIColor = .streamGrayDisabledText
```